integration/tests/epochs/suite.go

```diff
@@ -395,3 +387,18 @@ func (s *Suite) ExecuteReadApprovedNodesScript(ctx context.Context, env template
 
 	return v
 }
+
+func (s *Suite) WaitForEpochs(ctx context.Context, numOfEpochs int)  {
+	var epoch protocol.Epoch
+	var epochCounter uint64
+	for counter := 0; counter < numOfEpochs; counter++ {
+		// wait until the access node reaches the desired epoch
+		for epoch == nil || epochCounter != uint64(counter) {
```

Comment:
```suggestion
		for epoch == nil || epochCounter < uint64(counter) {
```
Terminate as soon as we reach any epoch at or past the target (avoids infinite loop, if the network is progressing).